### PR TITLE
Avatar new style

### PR DESCRIPTION
**Pull request by CarlSchwan**
Dark theme: image

White theme: 
image
image

Also fix a bug where the avatars where not generated with imagick because the font path was not correct 

Todos:
- [x] Add migration to delete all avatar with a generated files inside of them
- [x] Adapt vue lib to request dark themed image https://github.com/nextcloud/nextcloud-vue/pull/3119

### CODE REVIEW

**Review by jancborchardt — APPROVED** [review]
Looks real nice! :)

File: version.php [hunk 1]
```diff
@@ -30,7 +30,7 @@
 // between betas, final and RCs. This is _not_ the public version number. Reset minor/patchlevel
 // when updating major/minor version number.
 
-$OC_Version = [25, 0, 0, 7];
+$OC_Version = [25, 0, 0, 14];
```
Comment (CarlSchwan) [inline version.php:33]: Only for testing, need to be reset to 7 before merging

File: lib/private/Avatar/Avatar.php [hunk 2]
```diff
@@ -111,25 +111,27 @@ public function get(int $size = 64) {
...
 	 */
-	protected function generateAvatarFromSvg(int $size): ?string {
+	protected function generateAvatarFromSvg(int $size, bool $dark): ?string {
 		if (!extension_loaded('imagick')) {
 			return null;
 		}
 		try {
-			$font = __DIR__ . '/../../core/fonts/NotoSans-Regular.ttf';
-			$svg = $this->getAvatarVector($size);
+			$font = __DIR__ . '/../../../core/fonts/NotoSans-Regular.ttf';
```
[the remark this answers is not in the document]
Comment (CarlSchwan) [reply lib/private/Avatar/Avatar.php:133]: The path was wrong and then the imagick based avatar generation failed and used the slow gd fallback

**Review by come-nc — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Comment by CarlSchwan** [comment]
ci failure unrelated (sharing)

**Comment by ChristophWurst** [comment]
This should be documented so apps can adjust. I still see 

```json
{"reqId":"wfo0ZhmZNBDzXGliaakN","level":0,"time":"2023-01-26T09:32:37+00:00","remoteAddr":"127.0.0.1","user":"admin","app":"core","method":"GET","url":"/avatar/admin/32?v=4","message":"Avatar requested in deprecated size 32","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:108.0) Gecko/20100101 Firefox/108.0","version":"26.0.0.2","data":{"app":"core"}}
```

in my log when I open Dashboard

**Comment by ChristophWurst** [comment]
https://github.com/nextcloud/documentation/issues/9590

**Pull request merged**